tests_e2e/requirements.txt

```diff
@@ -0,0 +1,13 @@
+# This is a list of pip packages that will be installed on both the orchestrator and the test VM
```

Comment:
Requirements for the orchestrator/test VM. Comes directly from the 'dcr' POC, but should be splitted in 2 (one for orchestrator, one for test VM)